src/features/profile/frames/profile-frame.tsx

```
@@ -1,143 +1,176 @@
-import { Link } from '@tanstack/react-router';
-import { overlay } from 'overlay-kit';
 
 import { ProfileEditOverlay } from '../components/profile-edit-overlay';
 
-interface MenuButtonProps {
-  icon: React.ComponentType<{ className?: string }>;
-  onClick?: () => void;
-  variant?: 'default' | 'danger';
-  children: ReactNode;
+function formatDateTime(dateString: string) {
+  const { i18n } = useTranslation();
```

Biome (2.1.2)

[error] This hook is being called indirectly and conditionally, but all hooks must be called in the exact same order in every component render.

This is the call path until the hook.

Hooks should not be called after an early return.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)